report/chap/formalism.tex

```diff
@@ -1,209 +0,0 @@
-
-Not every game has a minimax equilibrium, however a weaker version of \eqref{eq:MinMax} (also called as $\epsilon$-equilibrium) holds for a large class of payoffs (see  \cite{roux_equilibria})
-\[
-    \inf_{\sigma \in \ST{0}} \sup_{\tau \in \ST{1}} f^v(\sigma, \tau) = \sup_{\tau \in \ST{1}} \inf_{\sigma \in \ST{0} } f^v(\sigma,\tau)
-\]
-
-For the games to be discussed, \eqref{eq:MinMax} will explicitly be shown. The minimax will exist from each $v \in V$.  Since $(\sigma^*,\tau^*)$ are history dependent they can be chosen independent of $v$. Hence we can bundle up the values and associate to $\G$ a value vector $\eta \in \Reals^V$ given by $\eta = (\eta_v)_{v\in V}$. Let $f(\sigma,\tau) = (f^v(\sigma,\tau))_{v \in V}$, then we have (inequalities are coordinate-wise) --
-\begin{equation}
-    \begin{aligned}
-    f(\sigma^*, \tau) \leq \eta \qquad \forall \tau \in \ST{1}\\
-    f(\sigma,\tau^*) \geq \eta \qquad \forall \sigma \in \ST{0}
-    \end{aligned}\label{eq:vecMinMax}
-\end{equation}
-
-\section{Finite Games}
-
-We will now start with a simple yet important class of payoffs, and show the existence of minimax and optimal strategies for it. 
-
-The payoff $f$ is finitely determined if there is a $N \in \Nat$ so that $f$ only depends on the outcomes in the first $N$ rounds. More precisely 
-\[
-    \forall \alpha,\beta \in \PI\, \alpha|_N=\beta|_N \implies f(\alpha) = f(\beta) 
-\]
-where $( v_0 v_1 \ldots )|_m = v_0v_1 \ldots v_m$.
-
-If $f$ finitely determined then call $\G=(G,f)$ a finite duration game. We can assume that the game stops after $N$ rounds have been played and the payoff is given by 
-\[
-    f : \PF[N] \to \Reals
-\]
-where $\PF[N]$ is the set of paths in $G$ of length (number of edges) $N$. This is defined as the payoff for of an arbitrary infinite extension, which is well defined since $G$ has no dead ends and $f$ depends only on first $N$ rounds.
-
-The following theorem is well known - it's sometimes called the backward induction technique. The backward induction, however, is not very explicit in the proof presented here.\\
-\begin{theorem}
-    \label{thm:finiteMinMax}
-    Let $\G=(G,f)$ be a finite game. There is a $\eta \in \Reals^V$ and strategies $(\sigma^*, \tau^*)$ which satisfy \eqref{eq:vecMinMax}
-\end{theorem}
-\begin{proof}
-    The proof is by induction on $N$ -- the duration of the game $\G$.
-    \begin{description}
-        \item[Base Case] $N=1$: Hence the game ends after the first move. 
-            \begin{align*}
-                \intertext{Let}
-                \eta_u &= \begin{dcases}
```

Comment:
This looks fine now.

On Wed, Jun 8, 2016 at 12:37 PM, Miheer Dewaskar <notifications@github.com>
wrote:

> In report/chap/formalism.tex
> <https://github.com/miheerdew/master-thesis/pull/2#discussion_r66202323>:
>
> > -    f : \PF[N] \to \Reals
> > -\]
> > -where $\PF[N]$ is the set of paths in $G$ of length (number of edges) $N$. This is defined as the payoff for of an arbitrary infinite extension, which is well defined since $G$ has no dead ends and $f$ depends only on first $N$ rounds.
> > -
> > -The following theorem is well known - it's sometimes called the backward induction technique. The backward induction, however, is not very explicit in the proof presented here.\\
> > -\begin{theorem}
> > -    \label{thm:finiteMinMax}
> > -    Let $\G=(G,f)$ be a finite game. There is a $\eta \in \Reals^V$ and strategies $(\sigma^*, \tau^*)$ which satisfy \eqref{eq:vecMinMax}
> > -\end{theorem}
> > -\begin{proof}
> > -    The proof is by induction on $N$ -- the duration of the game $\G$.
> > -    \begin{description}
> > -        \item[Base Case] $N=1$: Hence the game ends after the first move.
> > -            \begin{align*}
> > -                \intertext{Let}
> > -                \eta_u &= \begin{dcases}
>
> Sorry, I don't understand this comment.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/miheerdew/master-thesis/pull/2/files/21ac193b3378cb4de061fbcf78df9cfdb6ec025d#r66202323>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AS346O_d67i1pQ3Q5psh5KOEh1Wwtgezks5qJmorgaJpZM4IuINp>
> .
>